### ISSUE TITLE
Singleplayer in lobby can be run only once

### ISSUE THREAD

**Issue by ioma8** [ZeroK-RTS/Zero-K-Infrastructure#893]
Situation: I run singleplayer game (chicken), finish the game and then another singleplayer games can't be played. Clicking on the "Play" button does nothing except playing the "click" sound.
The only solution is to restart the lobby. So if I want to play 10 chicken single games in a row, I have to restart lobby 10 times.

This isssue had been there for me on two different computers over the last 1.5 years.
Both Windows 7, but different HW configuration.

**Comment by Licho1** [comment]
Hmm you heard click sound? What lobby program were you using Zero-K lobby?

**Comment by ioma8** [comment]
Yes, traditional Zero-K lobby, using the button in attached screen or in detail of the singleplayer game. I hear the click sound when I click, but no game starts, nothings happens. Works just the first time the lobby is run, than it has to be restarted to work again.
![image](https://cloud.githubusercontent.com/assets/6624752/11791266/8f9340e0-a2a0-11e5-9f86-5fb1d130fa55.png)

**Comment by knorker** [comment]
in taskmanager, do you maybe still see the process of the previous game?

**Comment by knorker** [comment]
http://code.google.com/p/zero-k/issues/detail?id=1946 is the same bug reported in Aug 21, 2013

One message by msafwan had some details:

> https://code.google.com/p/zero-k/source/detail?r=11690 :
workaround this issue for ZKL using internal browser.

> Usually it work like this:
1) user execute javascript (site_main.js, this file: https://code.google.com/p/zero-k/source/browse/trunk/other/Zero-K.info/Scripts/site_main.js )
2) then server send a message: SAIDPRIVATE Nightwatch !JSON SiteToLobbyCommand {"SpringLink":"@start_mission:Tutorial - Running Start r205"}
3) ZKL will do stuff to start mission.

> The problem is, the server never send 2nd message if you click the PLAY NOW 2nd time.

> The workaround is: internal browser read the URL for the javascript before its executed and run independently. This won't fix for user attempting mission/replay using external browser.
